portfolio/src/components/Gallery.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import closebtn from "../icons/png/001-cancel-3.png";
+import deletebtn from "../icons/png/delete.png";
+import Masonry, { ResponsiveMasonry } from "react-responsive-masonry";
+
+export default function Gallery({ galleryList }) {
+  const [isEditActive, setIsEditActive] = useState(false);
+
+  function handleOverlayActive() {
+    setIsEditActive((prevIsActive) => !prevIsActive);
```

Comment:
You need to pass to the handle the item id, so that its knows exactly which photo is active. 

So isActive in reality should be activeItemId. :) 

I hope this tip helps you

Reply:
It did, thanks! 😄